0000-template.md

```diff
@@ -1,94 +0,0 @@
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
```

Comment:
i dont' think you suppose to delete this file

Reply:
Oops I renamed the file

Reply:
Fixed